src/services/errors.js

```diff
@@ -22,5 +22,13 @@ module.exports= {
 	refreshTokenIsInvalidError : {
 		name : 'refreshTokenIsInvalidError',
 		message : "Refresh token is invalid."
+	},
+	inviteEmailDoesNotExistError : {
+		name: 'inviteEmailDoesNotExistError',
+		message: 'Email is not present in invites list'
+	},
+	userAlreadyActivatedError : {
+		name: 'userAlreadyActivatedError',
+		message: 'User is already activated'
```

Comment:
thinking through this, I think its basically a user creation endpoint so we can go with an error that says user already exists. I think that will be understood easier from the frontend

Kindly let me know if anything